spec/models/loan_health_check_spec.rb

```diff
@@ -154,6 +155,12 @@
 
               it { is_expected.to_not include :sporadic_loan_updates }
             end
+
+            context 'with no signing_date' do
+              let(:signing_date) { nil }
+
+              it { is_expected.to include :sporadic_loan_updates }
+            end
```

Comment:
Does this mean a loan with no signing date will always trigger the "Less than one step every month" warning?

Reply:
Hmm yeah maybe thats bad. Can you fix please? I am swamped.

Reply:
Actually no, it doesn't mean that. It is only triggering that warning because it has no future steps, and if there is no signing date, Then it defaults To the current date. If I move this Context block it will be clearer.